workflow-testing/src/test/java/com/squareup/workflow1/WorkerTest.kt

```diff
@@ -11,7 +11,7 @@ import kotlin.test.assertNotSame
 import kotlin.test.assertTrue
 
 /** Worker tests that use the [Worker.test] function. Core tests are in the core module. */
-class WorkerTest {
+internal class WorkerTest {
```

Comment:
Because we use explicit API mode, the IDE floods tests with warnings about the need to specificy visibility. Setting the class to `internal` squelches that noise and makes warnings useful again.